### PR TITLE
feat(quill): add quill composer

### PR DESCRIPTION
**Pull request by phatv**

### CODE REVIEW

File: src/components/ComposerQuill/composer.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+          defaultMenuOrientation: 'top',
+          mentionDenotationChars: ['@'],
+          source: this.handleMention,
+        },
+        toolbar: {
+          container: '#toolbar',
+        },
+      },
+      formats: ['mention'],
+      placeholder: 'Compose something awesome...',
```
Comment (andrewholsted) [inline src/components/ComposerQuill/composer.js]: 😂

File: src/components/ComposerQuill/composer.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      const text = this.quill.root.textContent.trim();
+      const parsed = md.renderInline(text);
+
+      send({displayName: text, content: parsed});
+      this.quill.setText('');
+    };
+  }
+
+  // Goes through the list and checks if the search term is in it
+  handleMention(searchTerm, renderList) {
```
Comment (andrewholsted) [inline src/components/ComposerQuill/composer.js]: This fine for a small list. As the list grows we will either need to figure out a way to dynamically ask for more, or optimize the search some.

**Comment by phatv** [comment]
What's the react storyboard?

**Comment by aimeex3** [comment]
you can run `yarn storyboard` and it will run the example app in index.stories.js. you can export multiple examples and it will have a tab for each one

**Pull request merged**